Dockerfile

```diff
@@ -97,11 +97,11 @@ USER root
 
 ### JAVASCRIPT
 
-# Install Node 14.0 and npm v7
+# Install Node 14.0 and npm v8
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - \
```

Comment:
Maybe this should be switched to node 16? This way we stick to the current LTS version, which also always include well-tested npm versions (no new features, only bugfixes).

You can use ``https://deb.nodesource.com/setup_lts.x`` to always point to the latest LTS or ``https://deb.nodesource.com/setup_16.x`` to just point to node 16.

This also removes the additional step of installing npm afterwards, which makes the docker building faster :)